tests/test_primonator.py

```diff
@@ -0,0 +1,28 @@
+from redirector.primonator import permalink_to_id, aleph_id_to_alma_id, alma_id_to_link
+
+
+def test_permalink_to_id():
+    assert permalink_to_id(None) is None
+    assert permalink_to_id(
+        'http://library.mit.edu/item/000192838') == '000192838'
+    assert permalink_to_id(
```

Comment:
I wanted a few around for future testing in case things get weird. These were all manually tested and confirmed to translate as expected. Relying on just one documented example could lead to that single record being deleted in the future and losing our only documented example. Not a huge deal, but that's what was in my brain :)